portal-web/test/functional/com/liferay/portalweb/macros/SearchAdministration.macro

```diff
@@ -1,23 +1,34 @@
 definition {
 
 	macro assertEngineVendorAndVersions {
-		AssertTextEquals.assertPartialText(
-			locator1 = "SearchAdmin#SEARCH_ENGINE_INFORMATION",
-			position = "1",
-			searchInformation = "${engineVendor}",
-			value1 = "${engineVendor}");
-
-		AssertTextEquals.assertPartialText(
-			locator1 = "SearchAdmin#SEARCH_ENGINE_INFORMATION",
-			position = "2",
-			searchInformation = "${clientVersion}",
-			value1 = "${clientVersion}");
-
-		AssertTextEquals.assertPartialText(
-			locator1 = "SearchAdmin#SEARCH_ENGINE_INFORMATION",
-			position = "3",
-			searchInformation = "${serverVersion}",
-			value1 = "${serverVersion}");
+		if (IsElementNotPresent(locator1 = "//div[contains(@class,'alert alert-info')]//strong[2]")) {
```

Comment:
why do we need these conditionals? does the UI vary between tests?

Reply:
Since the search admin portlet UI is different, ES7 and Solr have a connection list but ES 6 not, I uploaded them in attachments and left a comment on the ticket.